bedrock/settings/base.py

```diff
@@ -227,10 +232,24 @@ def lazy_langs():
 LANGUAGES = lazy(lazy_langs, dict)()
 
 FEED_CACHE = 3900
-DOTLANG_CACHE = 600
+DOTLANG_CACHE = config('L10N_CACHE_TIMEOUT', default='600', parser=int)
 
 DOTLANG_FILES = ['navigation', 'download_button', 'main']
 
+FLUENT_DEFAULT_FILES = ['navigation', 'footer', 'download_button']
```

Comment:
Side question: how do our default activation rules come into play with regard to global files? Do we have logic in place so these are always active?

Reply:
A comment is probably a good idea, and I can also just leave this list blank for now until we actually have these things converted.

Reply:
I tried to make the activation thing clear in the docs, so please let me know if I've not accomplished that. The way it works is that all of the .ftl files are evaluated for activation, but the only time that matters is if it's the file being used as the primary for the page. So even if a global file isn't "active" for a locale, if the one that's first in the list for the current page is active, then the page will be available regardless of the state of the other ftl files in the list passed to the view or the global list.

Reply:
A comment is probably a good idea, and I can also just leave this list blank for now until we actually have these things converted.

Reply:
Ok, that took a couple of seconds to sink in before I really grokked how activation effects these files, but I think your explaination already in the docs totally makes sense. My brain is still wired to how global files behave in the legacy system, and it may just take a while to override that :)